src/components/PopupDeleteCard.js

```diff
@@ -0,0 +1,31 @@
+import Popup from "./Popup.js";
+
+export default class PopupDeleteCard extends Popup {
+  constructor(popupSelector) {
+    super(popupSelector);
+    this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._popupDeleteButton =
+      this._popupElement.querySelector(".modal__button");
+    this._popupDeleteButtonText = this._popupDeleteButton.textContent;
+  }
+
+  handleDelete(handleDeleteSubmit) {
+    this._handleDeleteSubmit = handleDeleteSubmit;
+  }
+
+  renderLoading(isLoading, loadingText = "Saving...") {
```

Comment:
`"Saving..."`

In `PopupDeleteCard` you need to have `Deleting...` as a default value